libsoftwarecontainer/src/gateway/pulsegateway.cpp

```diff
@@ -23,20 +23,33 @@
 #include "pulsegateway.h"
 
 PulseGateway::PulseGateway() :
-    Gateway(ID)
+    Gateway(ID),
+    m_enableAudio(false)
 {
 }
 
 PulseGateway::~PulseGateway()
 {
 }
 
+bool PulseGateway::getConfValue(void)
+{
+    return m_enableAudio;
+}
```

Comment:
I don't like having this method only for being able to write tests. In that case it is better to break out activation into checking the value and calling another method for doing the work. Then, a test gateway could mock off the second method and could check that that second method is being called (because it will only be called if `m_enabled` is true)